packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -1,54 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { MDXRenderer } from 'gatsby-plugin-mdx';
-import styled from '@emotion/styled';
-import { ThemeProvider as UiKitThemeProvider } from 'emotion-theming';
+import { css } from '@emotion/core';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
-import SpacingsInline from '@commercetools-uikit/spacings-inline';
-import Stamp from '@commercetools-uikit/stamp';
 import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
 
-const DateElement = styled.div`
-  line-height: ${designSystem.typography.lineHeights.small};
-`;
-const Topics = styled.div`
-  color: ${designSystem.colors.light.textInfo};
-  font-size: ${designSystem.typography.fontSizes.small};
+import LayoutReleaseNoteBody from './layout-release-note-body';
+import Link from '../../components/link';
+
+const linkStyles = css`
+  text-decoration: none;
+  color: ${designSystem.colors.light.textPrimary} !important;
 
-  > * + * {
-    padding-left: ${designSystem.dimensions.spacings.xs};
-    margin-left: ${designSystem.dimensions.spacings.xs};
-    border-left: 1px solid ${designSystem.colors.light.surfaceSecondary3};
+  :hover {
+    color: ${designSystem.colors.light.linkNavigation} !important;
   }
 `;
-const stampTheme = {
-  fontSizeDefault: designSystem.typography.fontSizes.extraSmall,
-  // Override the `critical` style which is used for the "fix" type
-  colorError95: designSystem.colors.light.surfaceForReleaseNoteTypeFix,
-  colorError: designSystem.colors.light.borderForReleaseNoteTypeFix,
-};
 
 const ReleaseNote = (props) => {
   return (
     <SpacingsStack scale="m">
-      <Markdown.H3>{props.title}</Markdown.H3>
-      <DateElement>{props.date}</DateElement>
-      <UiKitThemeProvider theme={stampTheme}>
-        <SpacingsInline>
-          <Stamp tone={mapTypeToTone(props)}>{mapTypeToLabel(props)}</Stamp>
-        </SpacingsInline>
-      </UiKitThemeProvider>
-      {props.topics.length > 0 && (
-        <Topics>
-          {props.topics.map((topic) => (
-            <span key={topic}>{topic}</span>
-          ))}
-        </Topics>
-      )}
-
-      <div>
-        <MDXRenderer>{props.body}</MDXRenderer>
-      </div>
+      <Markdown.H3>
+        <Link css={linkStyles} href={props.slug}>
+          {props.title}
+        </Link>
```

Comment:
I thought @nkuehn mentioned that the titles in the overview page do not have links 🤔

Reply:
See the last item on the checklist here - https://github.com/commercetools/commercetools-docs-kit/issues/360

Reply:
@emmenko  I met with Sven over the complete "how do you link to a release note from where and how do you get the link" in a call this week and we came to the conclusion that we don't want two different link locations for the same release note in the world (one pointing to a position in the list and one to the details page). 

But if the headings in the list view don't have anchors, how do you get a link to a release note to send it to someone?   So we decided to show "read more" only if there is more, but let the heading be a link in any case.   Sven wanted the link to behave like in the footer or top nav. 

It's like it's done at GitHub - the release name is a link even if the details page does not contain more information.

Reply:
Ok thanks, would be nice if you guys can share this info next time.

Reply:
https://github.com/commercetools/commercetools-docs-kit/issues/360#issuecomment-614650409 

Sorry for not mentioning you so you get notified, I intended to but obviously didn't.